services/question.js

```diff
@@ -0,0 +1,18 @@
+const Question = require('~/models/question')
+
+const questionService = {
+  getQuestions: async (match, sort, skip = 0, limit = 10) => {
+    const items = await Question.collation({ locale: 'en', strength: 1 })
```

Comment:
I guess you firstly should find the Question:              
    const items = await Question.find(match)
      .collation({ locale: 'en', strength: 1 })
      .find(match)
      .sort(sort)
      .skip(skip)
      .limit(limit)
      .lean()
      .exec()